src/main/resources/messages/state-messages.properties

```diff
@@ -0,0 +1,47 @@
+ADD_END_DATE_STATE.invalid-date-format=Re-enter the date in the following format: {0}.
+ADD_END_DATE_STATE.cancel=Cancelled adding the end date.
+ADD_NAME_STATE.name-changed=Changed TODO name to {0}.
+ADD_NAME_STATE.new-todo-name=New TODO saved with name {0}. You can now edit.
+ADD_NAME_STATE.invalid-name=Description is too long or too short.
+ADD_NAME_STATE.cancel=Cancelled setting TODO name.
+ADD_START_DATE_STATE.no-todo-selected=No TODO selected, cancelling.
+ADD_START_DATE_STATE.todo-end-date-set=TODO end date {0} successfully set.
+ADD_START_DATE_STATE.invalid-date-format=Re-enter the date in the following format: {0}.
+ADD_START_DATE_STATE.cancel=Cancelled adding start date.
+ADD_TODO_NOTIFICATION_STATE.no-todo-selected=No TODO selected, cancelling.
+ADD_TODO_NOTIFICATION_STATE.todo-notification-time-set=You will be notified about this TODO at this time: {0}.
+ADD_TODO_NOTIFICATION_STATE.invalid-date-time-format=Re-enter the date and time in the following format: {0}.
+ADD_TODO_NOTIFICATION_STATE.cancel=Cancelled adding notification for TODO.
+BASE_STATE.active-todos-header=Active TODOs at this moment:
+BASE_STATE.select-todos-request=Select TODO (enter ID).
+BASE_STATE.enter-new-todo-name=Enter the name for your new TODO.
+BASE_STATE.enter-new-daily-notification-weekdays=Enter weekdays for the new notification (e.x. Mon, Monday).
+EDIT_TODO_STATE.cancel=Cancelled editing TODO.
+EDIT_TODO_STATE.no-todo-selected=No TODO selected, cancelling.
+EDIT_TODO_STATE.todo-saved=TODO saved.
+EDIT_TODO_STATE.enter-new-todo-name=Enter the new TODO name.
+EDIT_TODO_STATE.enter-todo-description=Enter the TODO description.
+EDIT_TODO_STATE.enter-start-date=Enter the start date and time of this TODO.
+EDIT_TODO_STATE.enter-end-date=Enter the end date and time of this TODO.
+EDIT_TODO_STATE.enter-notification-date-time=Enter the date and time when you need to be notified about this TODO.
+EDIT_TODO_STATE.daily-notification-set=Daily notifications for this TODO set to {0}.
+SELECT_DAYS_STATE.cancel=Cancelled creation of daily notification.
+SELECT_DAYS_STATE.invalid-weekdays-format=Write weekdays, separated with spaces (e.g. "Mon Tue Fri" or "Monday Tuesday Friday").
+SELECT_DAYS_STATE.weekdays-set=Weekdays selected, now write the time for notifications.
+SELECT_TIME_STATE.cancel=Cancelled creation of daily notification.
+SELECT_TIME_STATE.invalid-time-format=Re-enter the time in the following format: {0}.
+SELECT_TIME_STATE.time-set=You will receive notification on selected weekdays at {0}.
+SELECT_TODO_STATE.cancel=Cancelled selecting TODO.
+SELECT_TODO_STATE.invalid-id-format=Enter the number.
+SELECT_TODO_STATE.not-found=TODO with ID {0} not found.
+SELECT_TODO_STATE.selected=TODO {0} selected.
+TODO_SELECTED_STATE.cancel=Cancelled TODO selection.
+TODO_SELECTED_STATE.no-todo-selected=No TODO selected, cancelling.
+TODO_SELECTED_STATE.todo-marked-done=TODO is marked as done.
+TODO_SELECTED_STATE.start-editing=You can edit this TODO now.
```

Comment:
I hope most of them are autogenerated. Otherwise it would be imposible to maintain in the future...